index.toml

```diff
@@ -286,3 +286,14 @@ aliases = []
 completion_time = "15 min"
 created_at = 2023-11-30
 haystack_2 = true
+
+[[tutorial]]
+title = "Serialize Your Pipelines"
+description = "Learn how to serialize and deserialize your pieplines between YAML and Python"
+level = "beginner"
+weight = 6
```

Comment:
just curious - what does `weight` mean here?

Reply:
Ah, it's a Hugo thing, weight here is use to describe an overall order in which we want the tutorial to appear on the website, so I basically making this come _before_ the output validator one.. The reason for large/random numbers in the doc is because it means we can add stuff in between without having to edit everything.. But a better way would be nice